cmd/asset-syncer/postgresql_utils.go

```diff
@@ -116,14 +120,16 @@ func (m *postgresAssetManager) importCharts(charts []chart) error {
 		return err
 	}
 
-	err = txn.Commit()
-	if err != nil {
-		return err
-	}
+	return txn.Commit()
+}
 
-	// Remove charts no longer existing in index
+func (m *postgresAssetManager) removeMissingCharts(charts []chart) error {
+	var chartIDs []string
+	for _, chart := range charts {
+		chartIDs = append(chartIDs, fmt.Sprintf("'%s'", chart.ID))
+	}
 	chartIDsString := strings.Join(chartIDs, ", ")
-	_, err = m.db.Query(fmt.Sprintf("DELETE FROM %s WHERE info ->> 'ID' NOT IN (%s)", chartTable, chartIDsString))
+	_, err := m.db.Query(fmt.Sprintf("DELETE FROM %s WHERE info ->> 'ID' NOT IN (%s)", chartTable, chartIDsString))
```

Comment:
Throughout this file (not just this PR), we seem to be interpolating the query string ourselves, using `fmt.Sprintf`, which is dangerous? As per the signature of the methods in the interface, we should be passing the query string template *and* args to the `Query` command. For example, in this case,

```
_, err := m.db.Query(fmt.Sprintf("DELETE FROM %s WHERE info ->> 'ID' NOT IN (%s)", chartTable, chartIDsString))
```
should be
```
_, err := m.db.Query("DELETE FROM ? WHERE info ->> 'ID' NOT IN (?)", chartTable, chartIDs)
```
or similar?